ionic2/app/components/poke-details/poke-evolutions.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input } from '@angular/core';
+import { Pokemon } from '../../models/pokemon';
+import { ApiService } from '../../services/api.service';
+import { NavController } from 'ionic-angular';
+import { PokeDetailPage } from '../../pages/poke-detail/poke-detail.page';
+
+@Component({
+  template: require('./poke-evolutions.component.html'),
+  styles: [require('./poke-evolutions.component.scss')],
+  selector: 'poke-evolutions'
+})
+
+export class PokeEvolutionsComponent {
```

Comment:
Angular 2 Styleguide would ask you to import and implement the OnInit interface if you use ngOnInit